### ISSUE TITLE
Shadow of rose is not in first person

### ISSUE THREAD

**Issue by AndersMalmgren** [praydog/REFramework#581]
I bought shadow of rose (re8 DLC) and it's in third person instead of first.

**Comment by AndersMalmgren** [comment]
Tried the nighly build based on 627e6dbfb5f97bccf822a1e4a31b8efbde63d29f (RE8: fix)  Same problem.

**Comment by jossm87** [comment]
I try to play Shadow of Rose with the great VR Praydog RE8 mod in first person and I can't achieve It. Someone with success?

**Comment by xAstroBoy** [comment]
News about this?

**Comment by praydog** [comment]
No. If anything, you aren't going to see anything for a while. Nothing is planned at the moment.

Already busy with RE4 (a mainline entry, more important), and the UE injector.
